README.md

```diff
@@ -64,57 +64,3 @@ root directory.
 
 Here's a simple client program making a connection to Memgraph server and
 executing a single query:
-
-```c
-#include <stdio.h>
-#include <stdlib.h>
-
-#include <mgclient.h>
-
-int main(int argc, char *argv[]) {
-  if (argc != 4) {
-    fprintf(stderr, "Usage: %s [host] [port] [query]\n", argv[0]);
-    exit(1);
-  }
-
-  mg_session_params *params = mg_session_params_make();
-  if (!params) {
-    fprintf(stderr, "failed to allocate session parameters\n");
-    exit(1);
-  }
-  mg_session_params_set_host(params, argv[1]);
-  mg_session_params_set_port(params, (uint16_t)atoi(argv[2]));
-  mg_session_params_set_sslmode(params, MG_SSLMODE_REQUIRE);
-
-  mg_session *session = NULL;
-  int status = mg_connect(params, &session);
-  mg_session_params_destroy(params);
-  if (status < 0) {
-    printf("failed to connect to Memgraph: %s\n", mg_session_error(session));
-    mg_session_destroy(session);
-    return 1;
-  }
-
-  if (mg_session_run(session, argv[3], NULL, NULL) < 0) {
-    printf("failed to execute query: %s\n", mg_session_error(session));
-    mg_session_destroy(session);
-    return 1;
-  }
-
-  mg_result *result;
-  int rows = 0;
-  while ((status = mg_session_pull(session, &result)) == 1) {
-    rows++;
-  }
-
-  if (status < 0) {
-    printf("error occurred during query execution: %s\n",
-           mg_session_error(session));
-  } else {
-    printf("query executed successfuly and returned %d rows\n", rows);
-  }
-
-  mg_session_destroy(session);
-  return 0;
-}
-```
```

Comment:
Examples have to be referenced somehow from here. I think simple links are ok.

Reply:
The link should just be a github link to the folder? 
I'm not sure what would be the safest options for linking to something in the source code.

Reply:
The relative link should work [Example A](examples/example_a.c)... I think all rendering engines will correctly process that...

Reply:
+C++ example would also be useful :)

Reply:
Well, I'm just not sure for the first page of the project.
Relative links just modify the current path, if you have
https://github.com/memgraph/mgclient/tree/master and click the link it will go correctly to the 
https://github.com/memgraph/mgclient/tree/master/examples
but if you have https://github.com/memgraph/mgclient 
will that go correctly to the 
https://github.com/memgraph/mgclient/tree/master/examples
or
https://github.com/memgraph/mgclient/examples which doesn't exist?

Reply:
Let's try it out!